lib/db/patch.js

```diff
@@ -4,4 +4,4 @@
 
 // The expected patch level of the database. Update if you add a new
 // patch in the ./schema/ directory.
-module.exports.level = 39
+module.exports.level = 40
```

Comment:
FYI watch out for merge conflicts with https://github.com/mozilla/fxa-auth-db-mysql/pull/200